libs/@local/advanced-types/.babelrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't think we need this file

Reply:
I think you're right. It's still unclear to me which configs/scripts/tools we need to put into new packages and which we don't. It'd be really great if we had a guide (or even a command) on how to make a new package in our monorepo to be honest. cc @kachkaev

Reply:
Yeah I agree, that would be nice to have. I think this babelrc is for the block? I am actually not sure at all where babel is used :sweat_smile:

Reply:
I think I copied it from `isomorphic-utils`, so I'm not sure it's necessary for a block?